### PR TITLE
Add Memcached doc

### PR DESCRIPTION
**Pull request by ianmalott**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [x] Merge into master
- [x] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _services/memcached.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+## Service Creation
+
+You can create Memcached instances using the `convox services create` command:
+
+    $ convox services create memcached
+    Creating memcached-5864 (memcached)... CREATING
+
+This will provision Memcached on the Amazon ElastiCache. Creation can take a few minutes. To check the status use `convox services info <service-name>`.
```
Comment (mattmanning) [inline _services/memcached.md]: "the Amazon ElastiCache" sounds kind of funny. Can we drop the "the"?

File: _services/memcached.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+| `--instance-type=<instance.type>` | Elasticache instance type\* to use                  |
+| `--name=<name>`                   | Name of the service to create                       |
+| `--num-cache-nodes=<n>`           | Number of cache nodes the cache cluster should have |
+| `--private`                       | Create in private subnets                           |
+
+\* See [Available Elasticache node types](https://aws.amazon.com/elasticache/details/#Available_Cache_Node_Types)
+
+## Service Information
+
+To see relevant info about the database, use the `convox services info` command:
```
Comment (mattmanning) [inline _services/memcached.md]: Is "database" the right word here? Maybe "Memcached instance" or something?

File: _services/memcached.md (hunk 1 above)
Comment (ianmalott) [inline _services/memcached.md]: Yep, good catch. Copied this from the Redis doc and didn't think too much about that wording.

File: _services/memcached.md (hunk 2 above)
Comment (ianmalott) [inline _services/memcached.md]: Nope, "Memcached instance" makes more sense. Another copy-paste mistake.

**Pull request closed without merging**